chat-widget/automation_tests/e2e/utility/constants.ts

```diff
@@ -0,0 +1,130 @@
+
+  AdptivecardMsgAudio = "//div[@class='ac-textBlock']",
+  AdptivecardMsgThumbnailCard ="//div[@class='ac-textBlock']",
+  AdptivecardSigninCardMsg = "//div[contains(@class,'ac-container ac-adaptiveCard')]",
+  AdptivecardMsgReceiptCard = "//div[@class='ac-textBlock']",
+  AdptivecardThumbnailmsg = "//p[contains(text(),'Build and connect intelligent bots to interact with your users naturally wherever they are, from text/sms to Skype, Slack, Office 365 mail and other popular services.')]",
+  AdptivecardThumbnailimg = "//div[@class='ac-textBlock']/following::img[last()]",
+  AdptivecardGetstarted = "button[aria-label*='Get Started']",
+  AdptivecardReceiptcardOrderNo = "//p[contains(text(),'1234')]",
+  AdptivecardSigininCard = "button[aria-label*='Sign-in']",
+  AdptivecardMoreInformation = "button[aria-label*='More information']",
+  // Normal chat selectors
+  NormalChatLoading="//div[@id='oc-lcw-loading-pane']",
+  NoramlChatContainer="//*[@id='oc-lcw']//div[contains(@class,'webchat__basic-transcript')]",
+  NoramlChatAgentWithYouMessage="//*[@id='oc-lcw']//div[contains(text(),'An agent will be with you in a moment.')]",
+  NoramlChatClose="//button[@id='oc-lcw-header-close-button']",
+  NoramlChatCloseConfirmation="//div[@id='oc-lcw-confirmation-pane']",
+  NoramlChatConfirmCancel="//button[@aria-label='Cancel. Return to Chat']",
+  NoramlChatConfirmClose="//button[@aria-label='Close Chat']",
+  NoramlChatMinimize="//button[@title='Minimize']",
+  SpecialSymbolAsteriskMsg="//*[@id='oc-lcw']//div[text()='You said:']/following::div[@class='webchat__bubble__content']//p[text()='*Hi']/following::div[text()='Bot WC said:']/following::p[contains(text(),'*Hi')]",
+  SpecialSymbolPoundMsg="//*[@id='oc-lcw']//div[text()='You said:']/following::div[@class='webchat__bubble__content']//p[text()='#Hi']/following::div[text()='Bot WC said:']/following::p[contains(text(),'#Hi')]",
+  BotwaitGreetMsgwithHTMLLink = "https://www.microsoft.com/en-in/",
+  BotMessageExternalLink="//a[@title='Opens in a new window; external.']",
+  
+   //File attachment selctors
+   PrechatSurveyInput="//input[@aria-label='* Omnichannelsurvey']",
+   PrechatSurveySubmit="//button[@aria-label='Submit']",
+   UploadFile = "//*[@id='webchat']/div/div[5]/div/div/input|//button[@title='Upload file']/preceding::input[1]",
+   SendButton = "//button[contains(@title, 'Send')]",
+   UploadedFilename="//div[@class='webchat__fileContent__fileName']",
+   FileUpload="Fileupload.pdf",
+   IframelcwAttachement="//iframe[@id='Microsoft_Omnichannel_AMSClient_Iframe_Window']",
+   ImagePreview="//div[@aria-roledescription='attachment']//following::img",
+   UnsupportedFileTypeMsg="//div[contains(text(),'files are not supported.')]",
+   LimitExceedsMsg="//div[contains(text(),'File exceeds the allowed limit')]",
+}
+
+
+export enum Constants {
+    Two = 2,
+    Three = 3,
+    Five = 5,
+    MaxTimeout = 8000,
```

Comment:
move all timeout constants to TimeoutConstants

Reply:
Moved all time out constants to TimeoutConstants. Two, three and five are not time out constants.